### ISSUE TITLE
Opacity of YouTube video inside DHTML panel

### ISSUE THREAD

**Issue by jarari** [Facepunch/garrysmod-issues#2768]
### Details
I'm trying to make a youtube player that appears on user's screen, using dhtml panel and youtube iframe api.
When adjusting opacity of the video, I figured out that the video does not act as a child of the player.
In usual html5 browsers, contents inside div follows div's opacity like this: https://jsfiddle.net/ozthy95n/
But in Gmod, I couldn't figure out how to change the video's opacity.

Image1: With full opacity
![20160812204926_1](https://cloud.githubusercontent.com/assets/2539175/17621571/1f902068-60cf-11e6-8780-426ea9d8dc7f.jpg)

Image2:With about 10% opacity
![20160812204937_1](https://cloud.githubusercontent.com/assets/2539175/17621596/3fa4e474-60cf-11e6-9aef-32477d37359c.jpg)

Is it gmod's problem?

**Comment by Bo98** [comment]
It'll be because the Chromium engine in GMod is old and thus YouTube will use a Flash player rather than a HTML5 video player.

**Comment by jarari** [comment]
@Bo98 You're right. It works when I use iframe tag or embed tag with flash player element. 
But I have to make the player with new api since old one is deprecated.
